MAINTAINERS.md

```diff
@@ -0,0 +1,143 @@
+| Follow Contribution Guidelines                                                                                                              |            ✓            |            ✓           |            ✓            |
+| Enforce Contribution Guidelines                                                                                                             |            ✗            |            ✓           |            ✓            |
+| Co-own component or aspect of the library,<br>  including contributing: bug fixes, implementing features,<br> and performance optimizations |            ✗            |            ✓           |            ✓            |
+| Co-own on technical direction of component or<br> aspect of the library                                                                     |            ✗            |            ✗           |            ✓            |
+| Co-own on the project as a whole,<br> including determining strategy and policy for the project                                             |            ✗            |            ✗           |            ✓            |
+| _Privileges_                                                                                                                                |                         |                         |                         |
+| Permission granted                                                                                                                          |   [Read][permissions]   |   [Write][permissions]  | [Maintain][permissions] |
+| Eligible to become                                                                                                                          |       Code Owner        |       Maintainer        |            ✗            |
+| Can recommend <span style="color:Green">Contributors<br> to become Code Owner                                                               |            ✗            |            ✓           |            ✓            |
+| Can participate in promotions of<br> Code Owners and  Maintainers                                                                           |            ✗            |            ✓           |            ✓            |
+| Can suggest Milestones during planning                                                                                                      |            ✓            |            ✓           |            ✓            |
+| Can choose Milestones for specific component                                                                                                |            ✗            |            ✓           |            ✓            |
+| Paticipate in project's Milestones planning                                                                                                 |            ✗            |            ✗           |            ✓            |
```

Comment:
I am confused here a little bit. How can Contributor suggest milestones during planning but at the same time they cannot participate in project's planning?

Reply:
There is also a type:
`Paticipate` -> `Participate`

Reply:
> I am confused here a little bit. How can Contributor suggest milestones during planning but at the same time they cannot participate in project's planning?

Technically, during a PR submission a Contributor can say smth like "I'd like this fix to be included into the next oneTBB release".

Reply:
@omalyshe 
But it does not mean we will actually will do it for the next release.

Reply:
> @omalyshe But it does not mean we will actually will do it for the next release.

Yes, I agree with this. Suggestions from contributors will be considered but not necessarily included in the release.